src/components/StatusUpdater/index.tsx

```diff
@@ -114,7 +115,15 @@ const StatusUpdaterProtected = ({ activity, current, fullWidth, user, thisOrg }:
     setConfirming(true);
   }
 
-  const actions = getStatusOptions(current, activity.startTime, activity.forceStandbyOnly, activity.earlySignInWindow);
+  const lastOrgId = participant?.timeline[0].organizationId;
+  const currentOrgId = participant?.organizationId;
+  let actions;
+  if (currentOrgId !== lastOrgId) {
```

Comment:
This needs a condition on current status not equal to signed out or not responding.